src/utilities/valueFactory.js

```diff
@@ -20,25 +20,33 @@ const newValue = (
   component,
 })
 
-export const newLiteralValue = (property, literal, lang = defaultLanguageId) =>
+export const newLiteralValue = (property, literal, lang) =>
   newValue(property, literal, lang, null, null, null, "InputLiteralValue")
 
-export const newBlankLiteralValue = (property) =>
+export const newBlankLiteralValue = (property, propertyTemplate) =>
   newValue(
     property,
     null,
-    defaultLanguageId,
+    propertyTemplate.languageSuppressed ? null : Config.defaultLanguageId,
     null,
     null,
     null,
     "InputLiteralValue"
   )
 
-export const newUriValue = (property, uri, label, lang = defaultLanguageId) =>
+export const newUriValue = (property, uri, label, lang) =>
```

Comment:
Note: this method is always passed a language, so we do not need defaultLanguageId here